packages/opentelemetry-instrumentation-http/test/integrations/http-enable.test.ts

```diff
@@ -56,6 +57,43 @@ const customAttributeFunction = (span: Span): void => {
 };
 
 describe('HttpInstrumentation Integration tests', () => {
+  let mockServerPort = 0;
+  let mockServer: http.Server;
+  const sockets: Array<Socket> = [];
+  before(() => {
```

Comment:
```suggestion
  before((done) => {
```
`listen()` is async and `done` should be called once listen cb is invoked. This allows also to improve the error handling as throwing in listen callback is actually an unhandled exception.